app/controllers/katello/api/v2/content_export_incrementals_controller.rb

```diff
@@ -0,0 +1,99 @@
+module Katello
+  class Api::V2::ContentExportIncrementalsController < Api::V2::ApiController
+    before_action :fail_if_not_pulp3, :only => [:version, :library]
+    before_action :find_exportable_organization, :only => [:library]
+    before_action :find_exportable_content_view_version, :only => [:version]
+    before_action :find_library_export_view, :only => [:library]
+    before_action :find_history, :only => [:version, :library]
+
+    api :POST, "/content_export_incrementals/version", N_("Performs an incremental-export of a content view version.  Relevant only for Pulp 3 repositories")
+    param :id, :number, :desc => N_("Content view version identifier"), :required => true
+    param :destination_server, String, :desc => N_("Destination Server name, required for Pulp3"), :required => false
+    param :chunk_size_mb, :number, :desc => N_("Chunk export-tarfile into pieces of chunk_size mega bytes."), :required => false
+    param :from_history_id, :number, :desc => N_("Export history identifier used for incremental export. "\
+                                                 "If not provided the most recent export history will be used."), :required => false
+    def version
+      tasks = async_task(::Actions::Pulp3::Orchestration::ContentViewVersion::Export,
+                          content_view_version: @version,
+                          destination_server: params[:destination_server],
+                          chunk_size: params[:chunk_size_mb],
+                          from_history: @history)
+
+      respond_for_async :resource => tasks
+    end
+
+    api :POST, "/content_export_incrementals/library", N_("Performs an incremental-export of the repositories in library.  Relevant only for Pulp 3 repositories")
```

Comment:
I think I prefer omitting mention of pulp3 here and similarly across other APIs. We'll need to update this message, and it will need to be re-translated. As long as the API errors when called with a non-pulp3 API then maybe that's sufficient?

Reply:
Well I saw it as a part of hammer messaging.. But I am ok removing pulp3